src/Transport/Config/DefaultConfigurationValues.cs

```diff
@@ -34,7 +33,8 @@ void ApplyDefaultValuesForSerialization(SettingsHolder settings)
 
         void ApplyDefaultValuesForAddressing(SettingsHolder settings)
         {
-            settings.SetDefault(WellKnownConfigurationKeys.Topology.Addressing.Partitioning.Namespaces, new List<string>());
+            settings.SetDefault(WellKnownConfigurationKeys.Topology.Addressing.Partitioning.Namespaces, new NamespacesDefinition());
+            settings.SetDefault(WellKnownConfigurationKeys.Topology.Addressing.UseLogicalNamespaceName, (Func<NamespaceDefinition, string>)(x => x.ConnectionString));
```

Comment:
Because when I started working on this issue, I thought we need a way to switch between using name or connectionString. Through config API we setup the resolver func and use it in the transport avoiding `if` to check if feature has been enabled.

f.e. I thought to use this func to calculate the right value to put into ReplyTo header. If `UseLogicalNamespaceName`  feature has been enabled, ReplyTo header contains namespace name instead of connectionString. But now I'm struggling with the goal of the issue, because I noticed that ReplyTo header contains the queue name and it doesn't contain connectionString